llvm/Maintainers.rst

```diff
@@ -0,0 +1,467 @@
+================
+LLVM Maintainers
+================
+
+This file is a list of the
+`maintainers <https://llvm.org/docs/DeveloperPolicy.html#maintainers>`_ for
+LLVM.
+
+.. contents::
+   :depth: 2
+   :local:
+
+Current Maintainers
+===================
+The following people are the active code owners for the project. Please reach
+out to them for code reviews, questions about their area of expertise, or other
+assistance.
+
+**Warning: The maintainer list for LLVM is currently not up to date.**
+
+Lead maintainer
+---------------
+The lead maintainer is responsible for all parts of LLVM not covered by somebody else.
+
+| Chris Lattner
+| sabre\@nondot.org (email), lattner (GitHub), clattner (Discourse)
+
+
+
+Transforms and analyses
+-----------------------
+
+AliasAnalysis
+~~~~~~~~~~~~~~
+| Hal Finkel
+| hfinkel\@anl.gov (email), hfinkel (GitHub)
```

Comment:
Can you linkify the github names? I think you do this with a big regex, something like:
`s/, ([^ ]*) \(GitHub\)/, [\1](https://github.com/\1) (GitHub)/`
(untested)

Reply:
I've pushed the change for that, but I'm not sure I like the result. It does give us links in the rendered version, but it also makes the text file a good bit harder to read.

Reply:
For rst readability you could linkify the "GitHub" text. Also, sorry, as usual it's easy to get confused about when we're using rst or markdown. Since this is a new file, should we prefer Markdown, or is that premature?

Reply:
In https://llvm.org/docs/DeveloperPolicy.html#maintainers we mention Maintainers.rst, and this is what the clang and lldb CodeOwners.rst files currently use. Personally, I wouldn't mind switching this to markdown.

Reply:
I've pushed another commit converting the file to markdown, here's the result: https://github.com/nikic/llvm-project/blob/llvm-maintainers/llvm/Maintainers.md

Two things that were lost along the way is the table of contents, and the suppression of linkification for the emails (apparently this can't be cleanly disabled with markdown).